websites/K/KimCartoon/metadata.json

```diff
@@ -20,5 +21,13 @@
 		"videos"
 	],
 	"iframe": true,
-	"iFrameRegExp": ".*"
+	"iFrameRegExp": ".*",
```

Comment:
Anyway to improve? Probs not since they use a lot of services, but just checking...

Reply:
> Anyway to improve? Probs not since they use a lot of services, but just checking...

I'll look into this

Reply:
Changing this breaks the iframe part of the presence.